extras/kms/README.md

```diff
@@ -21,17 +21,30 @@ it:
   
   **IMPORTANT**: You should define a FK from `kms_root_key.scope_id` with
   cascading deletes and updates to the PK of the table within your domain that
-  tracks scopes.  This FK will prevent orphaned kms keys.
+  contains scopes.  This FK will prevent orphaned kms keys.
   
   For example, you could assign organizations and projects
   scope IDs and then associate a set of keys with each org and project within
   your domain. 
 
-- `root key`:  the KEKs (keys to encrypt keys) of the system.  
+- `root key`:  the KEKs (key-encryption-key) of the system.  
 
-- `data key`:  the DEKs (keys to encrypt data) of the system and must have a
+- `data key`:  the DEKs (data-encryption-key) of the system and must have a
   parent root key and a purpose.  
 
+- `data_key_version`: versions of DEKs (data-encryption-key) which are used to
```

Comment:
Why is this one snake case when the others are not?

Reply:
ty!